build/stf-run-ci/defaults/main.yml

```diff
@@ -9,6 +9,8 @@ __local_build_enabled: true
 __deploy_from_bundles_enabled: false
 __deploy_stf: true
 
+__service_telemetry_events_certificates_endpoint_cert_duration: 70080h
+__service_telemetry_events_certificates_ca_cert_duration: 2160h
```

Comment:
I'd expect this value to be selected when running ansible-playbook --extra-vars working_branch="vkmc-add-cert-duration-param" build/run-ci.yaml with no params, but it's defaulting to 70080h. I need to investigate why is that.